deployment/agent-deployment/ServiceGroupRoot/Parameters/ContainerInsights.Linux.Parameters.json

```diff
@@ -66,6 +42,18 @@
             {
               "name": "AGENT_IMAGE_FULL_PATH",
               "value": "__ACR_NAME__/public/azuremonitor/containerinsights/__AGENT_RELEASE__:__AGENT_RELEASE____AGENT_IMAGE_TAG_SUFFIX__"
+            }, 
+            {
+              "name": "CDPX_ACR",
+              "value": "__CDPX_LINUX_ACR__"
+            },
+            {
+              "name": "CDPX_REGISTRY",
+              "value": "__CDPX_LINUX_REGISTRY__"
+            },
+            {
```

Comment:
This tag should be extracted from linux-image-meta.json for linux and similarly for the windows, should be extracted from windows-image-meta.json.